base/applications/network/telnet/src/tnerror.cpp

```diff
@@ -81,6 +81,38 @@ int printit(const char * it){
 	return 0;
 }
 
+
+int wprintit(LPCWSTR it)
```

Comment:
I would suggest that, for this new function, you remove all these unnecessary comments in it.